minutes/2020-11-10.md

```diff
@@ -0,0 +1,52 @@
+# Parsec Community Meeting (10/11/2020) 📋
+
+Note: the content of this page will be recorded in the [community
+repository](https://github.com/parallaxsecond/community/tree/master/minutes).
```

Comment:
This doesn't usually go in the minutes, right?

Reply:
Not usually but every day is different 🤪

Reply:
No in reality I forgot to remove it from the template but I think it's fine to keep it